src/main/java/com/amazon/dlic/auth/http/jwt/keybyoidc/JwtVerifier.java

```diff
@@ -32,9 +32,11 @@ public class JwtVerifier {
 	private final static Logger log = LogManager.getLogger(JwtVerifier.class);
 
 	private final KeyProvider keyProvider;
+	private final int clockSkewToleranceSeconds;
 
-	public JwtVerifier(KeyProvider keyProvider) {
+	public JwtVerifier(KeyProvider keyProvider, int clockSkewToleranceSeconds ) {
```

Comment:
wdyt about passing this setting in directly into the call here: https://github.com/opensearch-project/security/blob/main/src/main/java/com/amazon/dlic/auth/http/jwt/AbstractHTTPJwtAuthenticator.java#L107 instead of constructing a verifier with this setting. The setting could then be passed into `validateClaims` where it is needed.

```
jwt = jwtVerifier.getVerifiedJwtToken(jwtString, clockSkewToleranceSeconds);
```

Reply:
The key provider is also used to validate the token and is currently passed in the constructor